tests/scenarios/test_functionality_nimbus.py

```diff
@@ -45,11 +49,25 @@ def test_browser_accepts_a_string():
     pass
 
 
+@scenario(
+    "../features/generic_functionality.feature",
+    "The browser will allow a new tab to be opened",
+)
+def test_browser_loads_a_new_tab():
+    pass
+
+
 @given("Firefox is launched enrolled in an Experiment")
 def selenium(selenium):
     return selenium
 
 
+@given("Firefox has loaded a webpage")
+def load_mozilla_wepage(navigate_using_url_bar, selenium):
+    navigate_using_url_bar()
+    assert "allizom" in selenium.current_url
```

Comment:
Can you explain this bit?
I'm guessing it works because allizom.org is the default domain when `text` is None and `use_clipboard` is false.
```py
    def _navigate_function(text=None, use_clipboard=False):
        if not text:
            text = "https://www.allizom.org/en-US/"
```

Reply:
That's correct!